src/android/ti/test.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * test module
+ *
+ * Created by Appcelerator, Inc
+ * Copyright (c) 2016 Appcelerator. All rights reserved.
+ */
+export { default as View } from './view';
+
+export default class Test {
+
+	constructor (app, module) {
+		this.app = app;
+		this.module = module;
+	}
+
+	startup () {
```

Comment:
When would this be called? Actually my question is more: is there anything a HL module will be able to do that HL code in the app itself cannot? Is this an example? That somehow modules will be able to run code before {{app.js}}?